packages/cysync-core/src/constants/i18n/en.ts

```diff
@@ -1384,6 +1384,323 @@ const en = {
       },
     },
   },
+  troubleShoot: {
+    createWallet: {
```

Comment:
Remove unused texts like `createWallet`

Reply:
Will update with the `settings` UI changes